<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>790</width>
    <height>457</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <widget class="QPushButton" name="pushButton_fileopen">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>打开文件</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_openselectfile">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>240</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>打开选中文件</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_searchsinglefile">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>100</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>查找单个文件</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_searchmiltfiles">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>140</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>查找多个文件</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_version">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>430</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>版本信息</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_close">
   <property name="geometry">
    <rect>
     <x>711</x>
     <y>430</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>关闭</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_status">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>430</y>
     <width>491</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>状态栏</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_sr_show">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>50</y>
     <width>381</width>
     <height>171</height>
    </rect>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget_filenames">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>281</width>
     <height>341</height>
    </rect>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::MultiSelection</enum>
   </property>
   <property name="resizeMode">
    <enum>QListView::Fixed</enum>
   </property>
   <property name="sortingEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget_sr_show">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>240</y>
     <width>381</width>
     <height>151</height>
    </rect>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::MultiSelection</enum>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_close</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>477</x>
     <y>391</y>
    </hint>
    <hint type="destinationlabel">
     <x>515</x>
     <y>322</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
